content/en/blog/falco-plugin-k8saudit-eks.md

```diff
@@ -51,68 +56,314 @@ With:
+{{% /pageinfo %}}
 
-Let's try with a dummy rule, just to check it works:
+We now have the plugins and rules in our system, the last part is to configure ` to use them.
+Modify your `/etc/yaml` to make it looks like this (with the relevant values for `REGION`, `ACCOUNT_ID` and `CLUSTER_NAME`):
 
 ```yaml
-- required_engine_version: 15
-- required_plugin_versions:
+rules_file:
+  ...
+  - /etc/k8s_audit_rules.yaml
+
+plugins:
   - name: k8saudit-eks
-    version: 0.1.0
+    library_path: libk8saudit-eks.so
+    init_config:
+      region: "{REGION}"
+      profile: "{AWS_PROFILE}"
+      shift: 10
+      polling_interval: 10
+      use_async: false
+      buffer_size: 500
+    open_params: "{CLUSTER_NAME}"
+  - name: json
+    library_path: libjson.so
+    init_config: ""
 
-- rule: Dummy rule
-  desc: >
-    Dummy rule
-  condition: >
-    ka.verb in (get,create,delete,update)
-  output: user=%ka.user.name verb=%ka.verb target=%ka.target.name target.namespace=%ka.target.namespace resource=%ka.target.resource
-  priority: WARNING
-  source: k8s_audit
-  tags: [k8s]
+load_plugins: ["k8saudit-eks","json"]
+...
 ```
 
+You can now restart Falco `sudo systemctl restart` and monitor the audit logs of your EKS cluster:
 ```shell
-falco -c falco.yaml -r rules/dummy_k8s_audit_rules.yaml 
+Wed May 10 14:07:18 2023: Falco version: 0.34.1 (x86_64)                                                                                                                                                     
+Wed May 10 14:07:18 2023: Falco initialized with configuration file: /etc/yaml                                                                                                                   
+Wed May 10 14:07:18 2023: Loading plugin 'k8saudit-eks' from file /usr/share/plugins/libk8saudit-eks.so                                                                                                
+Wed May 10 14:07:18 2023: Loading plugin 'json' from file /usr/share/plugins/libjson.so                                                                                                                
+Wed May 10 14:07:18 2023: Loading rules from file /etc/k8s_audit_rules.yaml                                                                                                                            
+Wed May 10 14:07:18 2023: Loading rules from file /etc/rules.d/override-k8saudit.yaml                                                                                                                  
+Wed May 10 14:07:18 2023: Starting health webserver with threadiness 4, listening on port 8765                                                                                                               
+Wed May 10 14:07:18 2023: Enabled event sources: k8s_audit                                                                                                                                                   
```

Comment:
Did you disable the syscall module? I don't see it in the config, but also don't see its output in the logs.